source/install/cloud-changelog.md

```diff
@@ -4,12 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-02-02](#release-2023-02-02)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
-- [Release 2022-11-17](#release-2022-11-17)
+
+## Release 2023-02-02
+
+### Highlights
+
+#### Annual Cloud Subscriptions
+ - On the purchase modal, admins are now able to buy an annual cloud subscription for a user count that exceeds the current user count.
+ - The **System Console > Billing & Account > Subscriptions** page now reflects whether the plan is monthly or annual.
+ - Added the ability to migrate from a monthly to an annual Cloud Professional plan.
```

Comment:
@johndavidlugtu should we also include a note that monthly is also retiring as a subscription option?

Reply:
Good callout @marianunez and yes

@amyblais Cloud Professional monthly will no longer be offered to new customers starting February 2. Suggest saying

- Cloud Professional monthly will no longer be offered to new customers starting February 2, 2023
- Added the option to migrate from a monthly to an annual Cloud Professional plan for existing Cloud Professional monthly customers